### ISSUE TITLE
add instructions for a signed ingress cert

### ISSUE THREAD

**Issue by mlbiam** [OpenUnison/openunison-k8s-login-activedirectory#14]
once deplyoed, its pretty easy to udpate but instructions would be good

**Comment by robertoayusob** [comment]
Hi Marc ¿what do I have to do to change the cert to put my own signed and avoid certificate problems?

**Comment by mlbiam** [comment]
The certificate is managed by the `Ingress` created, which references the secret `ou-tls-certificate` . Replace that secret with your signed certificate/keypair.

**Comment by robertoayusob** [comment]
And after that. Do I have to repeat the step the extract the certificate and dump to file to be used by apiserver (--oidc-ca-file=/etc/kubernetes/pki/ou-ca.pem)? true? And restart apiserver

**Comment by mlbiam** [comment]
It depends on if the cert is signed by a well known CA (ie LetsEncrypt) or not.  If it is, then you can remove the `--oidc-ca-file` parameter all together.

**Comment by mlbiam** [comment]
This is now taken care of via the operator and cr

**Comment by robertoayusob** [comment]
@mlbiam I have just read the issue https://github.com/kubernetes/kubernetes/issues/30650 and it looks I am in the same problem.

I am trying to create my own certificates for the openunison ingress. I have tried what I think that are all the posibilities:
1 use the apiserver certificate and key, overwrite ou-tls-certificate with them, and add the kubernetes ca to --oidc-ca-file
2 use custom ca certificates like current but created by myself, overwrite the  ou-tls-certificate with them and add the pem to the --oid-ca-file
3 use custom certificates signed by a custom ca, overwrite the ou-tls-certificate with them and add that ca to the --oid-ca-file parameter.

I tried to create a ca chain and add to the --oid-ca-file parameter.

Always the same result when trying to use the kubectl Command (kubernetes tokens) from ou web page, first requests are ok and after a pair of minutes I get this error on every kubectl command:

Unable to connect to the server: Get https://login......./auth/idp/k8sIdp/.well-known/openid-configuration: x509: certificate signed by unknown authority

Any Ideas?
